rask-engine/benches/bench.rs

```diff
@@ -0,0 +1,100 @@
+#![feature(test)]
```

Comment:
Should we add the benchmarking checks to our build-process (and CI)?

Also, please note that this only seems to work when running `cargo bench -p rask-engine`. Just running `cargo bench` breaks with the following error:

```
error: failed to run custom build command for `rask-wasm-shared v0.1.0 (/home/ma27/Projects/ratatosk/client/shared)`

Caused by:
  process didn't exit successfully: `/home/ma27/Projects/ratatosk/target/release/build/rask-wasm-shared-dd9268a7c49407aa/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'error: could not get or parse environment variable "MEMORY_LAYOUT_PARAMETERS": NotPresent', client/shared/build.rs:4:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

warning: build failed, waiting for other jobs to finish...
error: build failed
```

Reply:
Benching is currently not needed, as this is just a performance comparison.
`cargo bench` breaks, because it tries to build rask-wasm-shared without using the cargo make script.
This might be avoidable if the crate was excluded from test, or the workspace for that matter
Do you have any opinions on that?